src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -132,8 +117,40 @@ else if (!settingsDir.isDirectory())
     }
 
     /**
+     * Parse the options for the javadoc command line. The first item is the option name, and the
+     * following the number of items returned by {@link #optionLength(String)} for that option.
```

Comment:
Change this comment to something like:

Handle javadoc command line options. The first entry in the {@code options} array is the option name. Subsequent entries contain the option's arguments, the number of which is determined by the value returned from {@link #optionLength(String)} for that option.